doc/Building.md

```diff
@@ -0,0 +1,168 @@
+
+## Building from Source
+
+The standard build procedure using git and CMake is the following:
+
+    git clone https://github.com/BlueBrain/Tide.git
+    mkdir Tide/build
+    cd Tide/build
+    cmake ..
+    make
+
+## External Dependencies
+
+Tide builds on top of standard and well tested libraries for core
+functionalities such as rendering, serialization, file handling and network
+communication. Those dependencies are:
+* Boost 1.54 or later
+* MPI with MPI_THREAD_MULTIPLE support (openmpi 1.6.5 or later recommended)
+* Qt 5.4 or 5.5 (versions 5.6 and 5.7 currently have some
+  [stability issues](https://github.com/BlueBrain/Tide/issues/80))
+
+In addition, it also depends on some external projects that are automatically
+cloned by CMake during the configure step. They come with their own additional
+requirements:
+* Deflect: streaming of contents and applications
+  - libjpeg-turbo
+* TUIO: multitouch interface
+* VirtualKeyboard: on-screen virtual keyboard (Qml)
+* ZeroEQ: http / REST interface (technically optional, but still needed to
+  operate on-screen Launcher panel)
+  - Pthreads
+  - ZeroMQ
+
+Some additional libraries are searched by CMake to enable useful extra
+features if they are found:
+* Cairo + RSVG (>= 2.36.2): optimal %SVG support (QtSVG used as a fallback)
+* Cairo + poppler-glib: optimal PDF support
+* poppler-qt5: fallback PDF support
+* FFMPEG: playing movies
+* TIFF: display large images (TIFF image pyramids)
+
+### Automatic Package Installation
+
+To ease the process of searching and installing all the necessary system
+packages, an automatic installation process is provided. It is triggered by
+passing a specific argument to CMake during the configuration step.
+Currently, only Ubuntu (via apt-get) and OSX (via macports) are supported.
+
+To run the automatic install, do the following:
+
+    cd Tide/build
+    cmake .. -DINSTALL_PACKAGES=1
+
+One could also look directly at the TIDE_DEB_DEPENDS and TIDE_PORT_DEPENDS
+entries in the top-level CMakeLists.txt of each (sub)project and install the
+packages manually.
+
+## Supported Platforms
+
+This section gives more detailed information on building the software on some
+popular platforms.
+
+Unfortunately, many Linux distributions lack at least one of the required
+components which must be installed manually. New users are advised to consider
+Ubuntu 16.04 which is one of the simplest option overall.
+
+### Ubuntu 16.04
+
+Tide works almost out of the box on Ubuntu 16.04. The only issue is that the
+openmpi / mvapich packages are either buggy or lack support for
+MPI_THREAD_MULTIPLE.
+
+To boostrap the installation of Tide a on fresh install, do:
+
+    sudo apt install git cmake
+    git clone https://github.com/BlueBrain/Tide.git
+    mkdir Tide/build
+    cd Tide/build
+    cmake .. -DINSTALL_PACKAGES=1
+
+After this step all the necessary system packages should be installed. Then use
```

Comment:
Yes that would maybe be more intuitive, but then you need to install first the autotools etc. and I am not sure what exactly is needed. But if you run INSTALL_PACKAGES beforehand everything is there.